src/settings/Settings.ts

```diff
@@ -321,6 +321,11 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td("Show typing notifications"),
         default: true,
     },
+    "MessageComposerInput.ctrlEnterToSend": {
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        displayName: _td("Use Ctrl + Enter to send a message (Mac: Command + Enter)"),
```

Comment:
this string could vary depending on whether the device is Mac as the rest of the app does.

Reply:
What would be the correct way to make it platform specific? Simply: 
```
!isMac ? _td("PC string here") : _td("Mac string here")
```
I saw that in your PR you just called it 'Use Enter to send message'. However, I think it would be good to give indication how to send a message.

Reply:
Yup that looks correct.

Reply:
thanks fixed it